doc/reference/reference_lua/box_index/type.rst

```diff
@@ -0,0 +1,13 @@
+.. _box_index-type:
+
+===============================================================================
+index_object:type
```

Comment:
This should have a dot: `index_object.type`

Reply:
fixed in 22a984c